modules/extensions-soap-support/src/main/java/org/mule/runtime/module/extension/soap/internal/runtime/connection/ForwardingSoapClientConnectionProvider.java

```diff
@@ -82,4 +86,18 @@ public void disconnect(ForwardingSoapClient connection) {
   public ConnectionValidationResult validate(ForwardingSoapClient connection) {
     return success();
   }
+
+  @Override
+  public void initialise() throws InitialisationException {
+    if (transportProvider instanceof Initialisable) {
```

Comment:
should use `LifecycleUtils.initialiseIfNeeded(transportProvider, true, muleContext)` instead, since provider might require injection